cmd/glasskube/cmd/describe.go

```diff
@@ -35,6 +35,9 @@ var describeCmd = &cobra.Command{
 		fmt.Println(bold("Package:"), nameAndDescription(manifest))
 		fmt.Println(bold("Version:"), version(pkg, latestVersion))
 		fmt.Println(bold("Status: "), status(pkgStatus))
+		if pkg != nil {
+			fmt.Println(bold("Auto-Update:"), autoUpdateString(pkg))
```

Comment:
When auto updates are disabled, this just shows "Auto-Update: ", but I think in the describe command it would be better to show "Disabled". This is just my opinion, but I think it makes sense that in the list command we should only show "Enabled" but no "Disabled" (that way you can see the enabled ones faster), while in the describe command we show "Disabled". 

You could have an argument `disabledStr string` in the `autoUpdateString` function, such that the caller can define what should be shown when auto updates are disabled.